src/room.ts

```diff
@@ -132,4 +128,15 @@ export class Room {
 			userId: ''
 		};
 	}
+
+	private dealHand(conn: Connection, numCards: number) {
```

Comment:
Rename to `dealCards`. `dealHand` implies that resulting hand will have exactly `numCards` cards, though in reality you get `numCards` cards added to your existing hand.